api/inventorypb/agents.proto

```diff
@@ -185,6 +190,8 @@ message MongoDBExporter {
   string process_exec_path = 16;
   // Log level for exporter.
   LogLevel log_level = 17;
+  // Optionally expose the exporter process on 0.0.0.0
```

Comment:
I believe there should be better wording for the description. Maybe `expose exporter APIs to the world`